### ISSUE TITLE
Tips&tricks from the writer of securitynewsletter.co

### ISSUE THREAD

**Issue by treaz** [ericalexanderorg/SecurityBreach#75]
Note: this is just a dump from my email. We need to sort it out...

Some newsletters where I did a cross-promo with or sponsored them:
- tldr (really successful, made it into a permanent sponsorship once a month, always a nice bump in subscribers)
- morning cup of coding
- swift weekly
- data engineering weekly
- all the devops newsletters I could find, since there's a lot of overlap in interests. But most that I contacted have stopped I think.

Some security newsletters that I'll mention when I close the newsletter:
- https://nuzzel.com/InfoSecSherpa
- https://morninghacked.com/
- https://twitter.us18.list-manage.com/subscribe?u=e1ad6038c994abec17dafb116&id=a2457dc8ad (this week in security, great newsletter but apparently no real website for it)

I gathered breaches, and news in general, mostly from:
- https://www.bleepingcomputer.com/
- https://threatpost.com/
- https://www.zdnet.com/
- https://arstechnica.com/

Additionally: https://securitynewsletter.co/archive

**Comment by ericalexanderorg** [comment]
Will add a section for breach sources
